crates/biome_js_analyze/tests/specs/nursery/useSortedClasses/templateLiteralSpace.jsx.snap

```diff
@@ -43,17 +47,113 @@ templateLiteralSpace.jsx:3:15 lint/nursery/useSortedClasses  FIXABLE  ━━━
     2 │ 	<div class={`${variable} px-2 foo p-4 bar`}/>
   > 3 │ 	<div class={`px-2 foo p-4 bar ${variable}`}/>
       │ 	             ^^^^^^^^^^^^^^^^^
-    4 │ </>
-    5 │ 
+    4 │ 	<div class={`bar-${variable} mx-2 m-5`}/>
+    5 │ 	<div class={`mx-2 m-5 bar-${variable}`}/>
   
   i Unsafe fix: Sort the classes.
   
     1 1 │   <>
     2 2 │   	<div class={`${variable} px-2 foo p-4 bar`}/>
     3   │ - → <div·class={`px-2·foo·p-4·bar·${variable}`}/>
-      3 │ + → <div·class={`foo·bar·p-4·px-2·${variable}`}/>
-    4 4 │   </>
-    5 5 │   
+      3 │ + → <div·class={`foo·p-4·px-2·bar·${variable}`}/>
+    4 4 │   	<div class={`bar-${variable} mx-2 m-5`}/>
+    5 5 │   	<div class={`mx-2 m-5 bar-${variable}`}/>
+  
+
+```
+
+```
+templateLiteralSpace.jsx:4:30 lint/nursery/useSortedClasses  FIXABLE  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  ! These CSS classes should be sorted.
+  
+    2 │ 	<div class={`${variable} px-2 foo p-4 bar`}/>
+    3 │ 	<div class={`px-2 foo p-4 bar ${variable}`}/>
+  > 4 │ 	<div class={`bar-${variable} mx-2 m-5`}/>
+      │ 	                            ^^^^^^^^^
+    5 │ 	<div class={`mx-2 m-5 bar-${variable}`}/>
+    6 │ 	<div class={`mx-2 m-5 bar-${variable} px-2 p-5`}/>
+  
+  i Unsafe fix: Sort the classes.
+  
+    2 2 │   	<div class={`${variable} px-2 foo p-4 bar`}/>
+    3 3 │   	<div class={`px-2 foo p-4 bar ${variable}`}/>
+    4   │ - → <div·class={`bar-${variable}·mx-2·m-5`}/>
+      4 │ + → <div·class={`bar-${variable}·m-5·mx-2`}/>
+    5 5 │   	<div class={`mx-2 m-5 bar-${variable}`}/>
+    6 6 │   	<div class={`mx-2 m-5 bar-${variable} px-2 p-5`}/>
+  
+
+```
+
+```
+templateLiteralSpace.jsx:5:15 lint/nursery/useSortedClasses  FIXABLE  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  ! These CSS classes should be sorted.
+  
+    3 │ 	<div class={`px-2 foo p-4 bar ${variable}`}/>
+    4 │ 	<div class={`bar-${variable} mx-2 m-5`}/>
+  > 5 │ 	<div class={`mx-2 m-5 bar-${variable}`}/>
+      │ 	             ^^^^^^^^^^^^^
+    6 │ 	<div class={`mx-2 m-5 bar-${variable} px-2 p-5`}/>
+    7 │ </>
+  
+  i Unsafe fix: Sort the classes.
+  
+    3 3 │   	<div class={`px-2 foo p-4 bar ${variable}`}/>
+    4 4 │   	<div class={`bar-${variable} mx-2 m-5`}/>
+    5   │ - → <div·class={`mx-2·m-5·bar-${variable}`}/>
+      5 │ + → <div·class={`m-5·mx-2·bar-${variable}`}/>
+    6 6 │   	<div class={`mx-2 m-5 bar-${variable} px-2 p-5`}/>
+    7 7 │   </>
+  
+
+```
+
+```
+templateLiteralSpace.jsx:6:15 lint/nursery/useSortedClasses  FIXABLE  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  ! These CSS classes should be sorted.
+  
+    4 │ 	<div class={`bar-${variable} mx-2 m-5`}/>
+    5 │ 	<div class={`mx-2 m-5 bar-${variable}`}/>
+  > 6 │ 	<div class={`mx-2 m-5 bar-${variable} px-2 p-5`}/>
+      │ 	             ^^^^^^^^^^^^^
```

Comment:
As you might notice, the code action is correct, but the text range to the diagnostic is not. We need to calculate that as well